.github/workflows/github_actions.yml

```diff
@@ -55,7 +55,12 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        arch_with_features: [{arch: armv7, feature: none}, {arch: aarch64, feature: none}, {arch: aarch64, feature: crypto+crc}]
+        arch_with_features: [
+          {arch: armv7, feature: none, arch_cflags: none},
```

Comment:
At least one feature should be specified. That is `simd` (or NEON), right?

Reply:
Correct. 

Based on my experiment, `uraimo/run-on-arch-action` does not permit use to expand variables in `env` via any `if-else` expression. As a consequence, I use `none` to let `make` to fill the default value of `ARCH_CFLAGS` compile flags, just like the `FEATURE` variable in Makefile does.

Reply:
Add necessary comments to describe the features.

Reply:
Comments to describe the usage of `ARCH_CLFAGS` are added in README.